react/features/display-name/components/web/DominantSpeakerName.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+import { makeStyles } from '@material-ui/core/styles';
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { getDominantSpeakerParticipant, getLocalParticipant, getPinnedParticipant } from '../../../base/participants';
+import { withPixelLineHeight } from '../../../base/styles/functions.web';
+import { isToolboxVisible } from '../../../toolbox/functions.web';
+import { isLayoutTileView } from '../../functions';
+
+import DisplayNameBadge from './DisplayNameBadge';
+
+const useStyles = makeStyles(theme => {
+    return {
+        badgeContainer: {
+            ...withPixelLineHeight(theme.typography.bodyShortRegularLarge),
+            alignItems: 'center',
+            display: 'flex',
+            justifyContent: 'center',
+            marginBottom: theme.spacing(2),
+            transition: 'margin-bottom 0.3s'
+        },
+        containerElevated: {
+            marginBottom: theme.spacing(7)
+        }
+    };
+});
+
+const DominantSpeakerName = () => {
```

Comment:
docs: missing JSDoc

Reply:
Added.

Reply:
This looks a little bit weird. You changed the `DisplayNameBadge` to `function` but here you use `const`.

Could you please use single style for the PR? I don't mind if you use `const` as long as we have JSDocs for the component. My comment for the `DisplayNameBadge` was just a question, not really a change request.